modules/logging-loki-alerts.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+// logging-5-7-configuration
+
+:_content-type: PROCEDURE
+[id="logging-loki-alerts_{context}"]
+= Enabling log based alerts with Loki
+Loki alerting rules use link:https://grafana.com/docs/loki/latest/logql/[LogQL] and follow link:https://prometheus.io/docs/prometheus/latest/configuration/recording_rules/#recording-rules[Prometheus formatting]. You can set log based alerts by creating an `AlertingRule` custom resource (CR). `AlertingRule` CRs may be created for `application`, `audit`, or `infrastructure` tenants.
+
+[options="header"]
+|================================================
+| Tenant type    | Valid namespaces
+| application    |
+| audit          | `openshift-logging`
+| infrastructure | `openshift-/*`, `kube-/*`, `default`
+|================================================
+
+Application, Audit and Infrastructure alerts are sent to the Cluster Monitoring Operator (CMO) Alertmanager in the `openshift-monitoring` namespace by default, unless you have disabled the local `Alertmanager` instance.
+
+Application alerts are not sent to the CMO Alertmanager in the `user-workload-monitoring` namespace by default, unless you have enabled a separate `Alertmanager` instance.
+
+The `AlertingRule` CR is comprised of a set of specifications and webhook validation definitions to declare groups of alerting rules for a single LokiStack instance. In addition the webhook validation definition provides support for rule validation conditions:
+
+* If an `AlertingRule` CR includes an invalid `interval` period it is an invalid alerting rule
```

Comment:
Change it to as follows:

If an `AlertingRule` CR includes an invalid `interval` period, it is an invalid alerting rule